README.md

```diff
@@ -9,14 +9,6 @@ This repository contains the default client implementation.
 * For the windows native app, see https://github.com/secretin/secretin-windows
 * For the default server implementation, see https://github.com/secretin/secretin-server
 
-# Install
+Please read our wiki for installation guide : https://github.com/secretin/secretin-app/wiki
```

Comment:
Please leave the install process here. I don't want to open the wiki when I cloned it, and also, as it can change with time, it will be versioned with the app.